modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -417,8 +415,27 @@ class ONNXtest : public ::testing::Test {
         }
     }
 
-    void useModel(const std::string& model_name) {
+    // The function sets path to a model and a path to input test data,
+    // generates a cv::Mat if data is unreachable.
```

Comment:
Why does it make sense to generate cv::Mat if it can break `YOLOV3` tests as mentioned below ?

Reply:
I would chose single option:
* Image + skip tests if `opencv_extra` isn't available
* Random mat

Reply:
First option now.